BrainPortal/app/controllers/portal_controller.rb

```diff
@@ -46,6 +46,15 @@ def welcome #:nodoc:
     @default_data_provider  = DataProvider.find_by_id(current_user.meta["pref_data_provider_id"])
     @default_bourreau       = Bourreau.find_by_id(current_user.meta["pref_bourreau_id"])
 
+    @tool_names = Tool
+      .joins(:tool_configs)
+      .where.not(category: 'background')
+      .where('tool_configs.bourreau_id IS NOT NULL')
+      .where('tool_configs.bourreau_id > 0')
+      .where('tool_configs.version_name IS NOT NULL')
+      .distinct
+      .pluck 'name'
```

Comment:
Last change: can we get this list sorted? The list was sorted in the original page, but this query doesn't get it sorted.